boxoffice/templates/cash_receipt_new.html.jinja2

```diff
@@ -0,0 +1,557 @@
+                <p>{%- trans %}Phone {% endtrans -%}</p>
+                <p>{{ order.buyer_phone }}</p>
+              </td>
+            </tr>
+          </table>
+        </div>
+      <!--[if (gte mso 9)|(IE)]>
+      </td><td valign="top" style="width:300px;">
+      <![endif]-->
+        <div style="display:inline-block;vertical-align:top;width: auto">
+          <table>
+            <tr>
+              <td class="invoice-label">
+                <p>{%- trans %}Paid on {% endtrans -%}</p>
+                <p>{{ order.paid_at | longdate }}</p>
+              </td>
+            </tr>
+            <tr>
+              <td class="invoice-label">
+                <p>{%- trans %}Email {% endtrans -%}</p>
+                <p>{{ order.buyer_email }}</p>
+              </td>
+            </tr>
+          </table>
+        </div>
+      <!--[if (gte mso 9)|(IE)]>
+      </td>
+      </tr>
+      </table>
+      <![endif]-->
+    </td>
+  </tr>
+  <tr align="left">
+    <td class="invoice-header" colspan="4">
+      <p>{%- trans %}Order Summary{% endtrans -%}</p>
+    </td>
+  </tr>
+    <tr align="left" class="order-summary">
+      <th colspan="3">{% trans %}Item{% endtrans %}</th>
+      <th align="right">{% trans %}Price{% endtrans %}</th>
+    </tr>
+    {% for line_item in line_items -%}
+      <tr>
+        <td colspan="3" class="order-description">
+          <p class="item-title">{{ line_item.ticket.title }}</p>
+          <p class="item-description">{{ line_item.ticket.description| safe}}</p>
+        </td>
+        <td align="right" valign="top" class="base-price">
+          <p>{{ currency_symbol }} {{ line_item.base_amount }}</p>
+        </td>
+      </tr>
+      <tr>
+        <td colspan="4"><hr/><br/></td>
+      </tr>
+      <tr align="right">
+        <td colspan="4">
+          <table style="width:100%">
+            <tr align="right">
+              <td colspan="3">
+                <p>{% trans %}Sub-total{% endtrans %}</p>
+              </td>
+              <td>
+                <p>{{ currency_symbol }} {{ line_item.base_amount }}</p>
+              </td>
+            </tr>
+            <tr align="right">
+              <td colspan="3">
+                <p>{% trans %}Discount{% endtrans %}</p>
+              </td>
+              <td>
+                <p>{{ currency_symbol }} {{ line_item.discounted_amount }}</p>
+              </td>
+            </tr>
+            <tr align="right">
+              <td class="total-price" colspan="3">
+              <p>{% trans %}Total Paid{% endtrans %}</p>
+              </td>
+              <td class="total-price">
+                <p>{{ currency_symbol }} {{ line_item.final_amount }}</p>
+              </td>
+            </tr>
+          </table>
+        </td>
+      </tr>
+    {%- endfor %}
+    <tr>
+      <td colspan="4">
+        <p>*To access the benefits of this ticket please ensure you are logged in to your Hasgeek account with this same email address.</p>
```

Comment:
`{% trans %}` tag needed here. The leading `*` isn't necessary as this is not a footnote.